src/components/dace_interface.ts

```diff
@@ -177,12 +200,22 @@ export class DaCeInterface {
         return vscode.Uri.joinPath(extensionUri, 'backend', 'run_dace.py');
     }
 
-    public async startDaemonInTerminal(): Promise<void> {
+    @ICPCRequest()
+    public async quitDaemon(): Promise<void> {
+        this.daemonTerminal?.dispose();
+        this.daemonTerminal = undefined;
+        this.daemonBooting = false;
+        this.daemonRunning = false;
+        this.invoke('setStatus', [false]);
+    }
+
+    @ICPCRequest()
+    public async startDaemonInTerminal(port?: number): Promise<void> {
```

Comment:
Is there a way to set this? e.g., in settings?

Reply:
Done, now settings overrideable